dessia_common/serialization.py

```diff
@@ -715,25 +760,31 @@ def pointers_analysis(obj):
             if path2 in class_from_path:
                 val2_class = class_from_path[path2]
             else:
-                val2 = get_in_object_from_path(obj, path2)
-                val2_class = dcty.full_classname(val2)
-                class_from_path[path2] = val2_class
+                try:
```

Comment:
Note for later : we should name val1_class and val2_class with more explicit names. I personnally like the "object / other_object" convention